.cirrus.yml

```diff
@@ -148,7 +149,12 @@ task:
         # CPUs, and needed at least 10G of RAM to not run out of memory.
         CPU: 4
         MEMORY: 10G
+        SHOULD_UPDATE_PACKAGES: "FALSE"
       script:
+        - (cd packages/flutter_tools; pub get)
+        - (cd packages/flutter_tools/test/data/asset_test/main; pub get)
```

Comment:
This is definitely something we could clean up to, by just checking in a handwritten package config.

Reply:
(not now)